concepts/lists/about.md

```diff
@@ -47,7 +47,7 @@ List.length [ 7, 8 ] --> 2
 Any function/operator that appear to modify a list (such as adding an element), will actually return a new list.
 Performance is usually not an issue though, as the implementation of lists prevents unnecessary allocations/copies.
 
-Lists can also be processed using [pattern matching](list-destructuring).
+Lists can also be processed using [pattern matching][list-destructuring].
```

Comment:
It seems to work fine though...

<img width="794" alt="Screenshot 2024-03-07 at 21 21 01" src="https://github.com/exercism/elm/assets/16929078/1aa33168-9ffb-454b-b159-890604d269c2">

Reply:
Seems, yes. Did you click it?

Reply:
Woah, you just blew my mind, I can't believe I didn't click it :D

In that case, there is at least one more I would like you to fix here: `exercises/concept/tracks-on-tracks-on-tracks/.docs/introduction.md`

Reply:
I've found a couple more by looking at the output at `grep '](' **/.*/**/*.md` and `grep '](' **/*.md`. Could you get them too?

```
exercises/concept/top-scorers/.docs/hints.md:- You will need to import the [Dict module](dict-module).
exercises/concept/tracks-on-tracks-on-tracks/.docs/hints.md:- There are [various ways to combine lists](combine).
```

Reply:
Done.